.github/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+<!--- indicate the protocol code in the Title above -->
+
+## Task list
+
+<!--see https://docs.github.com/en/github/managing-your-work-on-github/about-task-lists
+for an explanation on how to use task lists-->
+
+Steps by contributor:
+
+- [ ] Add description to this pull request (see #-description)
+- [ ] Submit the pull request by clicking 'create pull request'
+- [ ] Mark the pull request as draft
+- [ ] Add further commits and push commits to GitHub if needed
+- [ ] Update the protocol-specific NEWS.Rmd
```

Comment:
```suggestion
- [ ] Update the protocol-specific NEWS.md
```

Reply:
It's an Rmd file [see template](https://github.com/inbo/protocolhelper/tree/master/inst/rmarkdown/templates/template_sfp_nl/skeleton)

Reply:
Can you remind my why we have it as an Rmd? If it doesn't contain R code, i'd use an md. Because that can be rendered by GitHub.